web/src/main/java/com/bazaarvoice/emodb/web/resources/sor/DataStoreResource1.java

```diff
@@ -776,6 +776,25 @@ public SuccessResponse updateAllForFacade(InputStream in, @QueryParam ("tag") Li
         return SuccessResponse.instance();
     }
 
+    @POST
+    @Path("_updateRef")
```

Comment:
blocker: Why is the path has '_' prefix? This is not aligned with API Guidelines - https://backstage.prod.bazaarvoice.com/docs/default/component/api-guidelines/urls/#must-use-kebab-case-for-path-segments-129.
Even if its an internal only API, do follow API-Guidelines

Reply:
endpoint has been modified to `{channel}/sendbatch1` after discussion with bharadwaj